assignments/week-01/eval.go

```diff
@@ -0,0 +1,60 @@
+package main
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"errors"
+	"regexp"
+)
+
+func eval(text string) error {
+
+	s := strings.Split(trim(text)," ")
+	if (len(s)<3){
+		return errors.New("Too few arguments")
+	}
+
+	if (len(s)>3){
+		return errors.New("Too many arguments")
+	}
+
+	t1,op,t2 := s[0], s[1], s[2]
+	n1, errN1 := strconv.Atoi(t1)
```

Comment:
You should return if `errN1` is not `nil` so that we don't need to run line 23.

Reply:
thanks